core/logger/logger.go

```diff
@@ -262,7 +270,7 @@ func (c Config) DebugLogsToDisk() bool {
 
 // RequiredDiskSpace returns the required disk space in order to allow debug logs to be stored in disk
 func (c Config) RequiredDiskSpace() utils.FileSize {
-	return utils.FileSize(c.FileMaxSize * (c.FileMaxBackups + 1))
+	return utils.FileSize(c.FileMaxSize * utils.MB * (c.FileMaxBackups + 1))
```

Comment:
This is still confusing me, because `FileMaxSize` is already a `utils.FileSize`, which says it is in bytes:
```go
// FileSize repesents a file size in bytes.
type FileSize uint64
```

Reply:
Oh, now I see this:
`FileMaxSize    int // megabytes`
Why don't we suffix the field name with the units so that it's clear at the call sites? e.g. `FileMaxSizeMB int`?

Reply:
Alternatively, we could pass along `utile.FileSize` for better precision, and only scale to MB at the end.

Reply:
Fixed